code/modules/martial_arts/martial.dm

```diff
@@ -74,13 +118,13 @@
 		D.forcesay(GLOB.hit_appends)
 	else if(D.lying)
 		D.forcesay(GLOB.hit_appends)
-	return 1
+	return TRUE
 
-/datum/martial_art/proc/teach(var/mob/living/carbon/human/H,var/make_temporary=0)
-	if(help_verb)
-		H.verbs += help_verb
+/datum/martial_art/proc/teach(mob/living/carbon/human/H, make_temporary = FALSE)
```

Comment:
Todo: call remove on the previous martial arts and make sure the temp arts are handled correctly